src/dbnode/storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -240,7 +241,8 @@ func (s *peersSource) readData(
 
 		s.log.Info("peers bootstrapper resolving block retriever", zap.Stringer("namespace", namespace))
 
-		r, err := retrieverMgr.Retriever(nsMetadata)
+		// TODO(bodu): Discuss right approach here.
```

Comment:
@robskillington need to discuss the call below.

Reply:
Let's try to rip this out for now as discussed.